sql-statements/sql-statement-cancel-import-job.md

```diff
@@ -7,10 +7,6 @@ summary: An overview of the usage of CANCEL IMPORT in TiDB.
 
 The `CANCEL IMPORT` statement is used to cancel a data import job created in TiDB.
 
-> **Note:**
->
-> This feature is not available on [TiDB Cloud Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-cloud-serverless) clusters.
-
```

Comment:
Please check if the same note on https://docs.pingcap.com/tidb/dev/sql-statement-show-import-job/ also needs to be removed.